lib/git.js

```diff
@@ -9,12 +10,14 @@ var urlFormat = require('url').format;
 
 function getCurrentBranch(workingDir) {
   if (!shell.pushd(workingDir)) {
-    return Error(util.format('Directory %s does not exist', workingDir));
+    return Error(GLB.t('msgDirectoryDoesntExist', {
+      phWorkingDir: workingDir,
+    }));
   }
   var output = shell.exec('git symbolic-ref --short HEAD', {silent: true});
   shell.popd();
   if (output.code !== 0) {
-    return Error('This directory does not contain a valid git repository');
+    return Error(GLB.t('msgDirectoryNoGitRepo'));
```

Comment:
String extraction is done offline, not runtime, to the string resource file so that we can hand-off to the human translator.